packages/ifpack2/test/unit_tests/Ifpack2_UnitTestSGSMT.cpp

```diff
@@ -120,6 +120,33 @@ TEUCHOS_UNIT_TEST_TEMPLATE_3_DECL(SGS_MT, JacobiComparison, Scalar, LO, GO)
   X_diff_mtgs.update (1, X_wanted, -1);
   double normInf_mt_gs = X_diff_mtgs.normInf ();
 
+
+  // Test two-stage Symmetric Gauss-Seidel (SGS) with three sweeps.

```

Comment:
It looks like there's 10 outer and 100 total inner sweeps, can you update this comment?